### PR TITLE
docs(service-worker): add doc to include updates to the index.html with the new ng add command

### PR DESCRIPTION
**Pull request by sreevani-ship-it**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Currently, the doc does not have content that explains updates to the index.html with the `ng add  @angular/pwa` command.
Issue Number: 23373 https://github.com/angular/angular/issues/23373

## What is the new behavior?
Added doc to explain the updates to index.html file with the `ng add  @angular/pwa` command

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Brocco — APPROVED** [review]
LGTM

File: aio/content/guide/service-worker-getting-started.md [hunk 1]
```diff
@@ -23,10 +23,14 @@ ng add  @angular/pwa --project *project-name*
...
 
-1. Adds the `@angular/service-worker` package. 
+1. Adds the `@angular/service-worker` package to your project. 
 2. Enables service worker build support in the CLI.
 3. Imports and registers the service worker in the app module.
-4. Creates the service worker configuration file called `ngsw-config.json` which specifies the caching behaviors and other settings. 
+4. Updates the `index.html` file:
+    * Includes a link to add the `manifest.json` file.
+    * Adds meta tags for `theme-color`.
+5. Installs icon files to support the installed Progressive Web App (PWA).
```
Comment (jenniferfell) [inline aio/content/guide/service-worker-getting-started.md]: Unclear reference: We've never mentioned "the installed Progressive Web App (PWA)" using that phrase before. Is this the app/project to which we're currently adding service worker support? 

And I don't think we've been clear in the past about what icon file we're adding any why.  What's the icon file for? Also....as a developer, I might wonder why you're adding icon files to my app? Is this the file we're talking about: favicon.ico?

File: aio/content/guide/service-worker-getting-started.md [hunk 2]
```diff
@@ -23,10 +23,14 @@ ng add  @angular/pwa --project *project-name*
...
+6. Creates the service worker configuration file called `ngsw-config.json` which specifies the caching behaviors and other settings.
```
Comment (jenniferfell) [inline aio/content/guide/service-worker-getting-started.md]: Missing comma: Creates the service worker configuration file called `ngsw-config.json`, which specifies the caching behaviors and other settings. 

Also...maybe add a link from this filename to the doc file about config options?

**Review by jenniferfell — CHANGES_REQUESTED** [review]
Please clarify which "installed PWA" because we haven't used that phrase in these docs before. Other comments are optional, but while you're here... :-)

File: aio/content/guide/service-worker-getting-started.md [hunk 3]
```diff
@@ -23,10 +23,14 @@ ng add  @angular/pwa --project *project-name*
...
 The above command completes the following actions:
 
-1. Adds the `@angular/service-worker` package. 
+1. Adds the `@angular/service-worker` package to your project. 
 2. Enables service worker build support in the CLI.
 3. Imports and registers the service worker in the app module.
-4. Creates the service worker configuration file called `ngsw-config.json` which specifies the caching behaviors and other settings. 
+4. Updates the `index.html` file:
+    * Includes a link to add the `manifest.json` file.
+    * Adds meta tags for `theme-color`.
```
[the remark this answers is not in the document]
Comment (chembu) [reply aio/content/guide/service-worker-getting-started.md]: I checked with him and he confirmed that the Description meta tag is not added.

**Review by jenniferfell — APPROVED** [review]

**Comment by jenniferfell** [comment]
I still think we can be more clear about this item:
"Installs icon files to support the installed Progressive Web App (PWA)."

But we can come back as necessary if others are confused. Nothing technically incorrect or missing.

**Comment by jenniferfell** [comment]
@IgorMinar Hi. The essentials have been updated. With a pull-approve member approval, I think we can merge.

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**